view-creator-framework/build.gradle.kts

```diff
@@ -71,6 +73,8 @@ dependencies {
   annotationProcessor("org.projectlombok:lombok:1.18.26")
   implementation("org.slf4j:slf4j-api:2.0.7")
   implementation("com.typesafe:config:1.4.2")
+  implementation("io.airlift:aircompressor:0.27")
+  implementation("io.netty:netty-codec-http:4.1.108.Final")
```

Comment:
Netty packages should be all upgraded together - to do that, we use the netty bom which is already declared on line 68. Upgrade the version there instead.